ashahba/Start to upgrade r0.19 branch to TF v1.15.0 (#431)
tensorflow/ngraph-bridge: test/python/tensorflow/python_tests_list.txt

#softmax_op_test.SoftmaxTest.testHalfGPU
softmax_op_test.SoftmaxTest.testLargeDims

> # Test failes due to DT_INT8 not implemented during TF v1.15.0 upgrade

Reviewer: ```suggestion
# Test fails due to DT_INT8 not being implemented for TF v1.15.0
```
